packages/debug/src/browser/style/breakpoint-function-disabled.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@tolusha I've copied breakpoing svg files also from VS Code. CQs were approved already, but please add copyrights to them 🙏

Reply:
Done